pkg/cli/resources.go

```diff
@@ -13,6 +14,10 @@ import (
 	"github.com/convox/stdcli"
 )
 
+const (
+	RESOURCE_URL_PATTERN = `.+:\/{2}.+`
```

Comment:
what pattern it solves
add example or unit test on that

Reply:
I added the unit test, I'll add a comment above explaining it.